cli.py

```diff
@@ -1,11 +1,31 @@
 """Main file to call universions from the CLI."""
 
 import argparse
+import pkg_resources
 
+import universions
 import universions.java as uj
 import universions.node as un
 
-TOOLS = {"java": uj.get_java_versions, "node": un.get_node_versions}
+
+def get_self_version():
+    """
+    Gets the version of this exact module.
+    From the advised alternatives in
+    https://packaging.python.org/guides/single-sourcing-package-version/, this reads from setup.py.
+    But there is a little problem: for a version such as 0.1.0-dev21, the attribute version returns
+    '0.1.0.dev21'. This issue is that the '-' is replaced by a '.', which makes it an invalid semver
+    """
+    raw_version = pkg_resources.require("universions")[0].version
+    version = raw_version.replace(".dev", "-dev")
+    return universions.parse_semver(version)
```

Comment:
In latest version you can use `universions.__version__`

I think python requires the libraries version to be `.devN`, which is bad for SemVer. I need to investigate more.

Reply:
Unfortunately not working with python 3.7.4 😞